.travis.yml

```diff
@@ -22,7 +22,7 @@ matrix:
   include:
     # This environment tests that scikit-learn can be built against
     # versions of numpy, scipy with ATLAS that comes with Ubuntu Trusty 14.04
-    - env: DISTRIB="ubuntu" PYTHON_VERSION="2.7" CYTHON_VERSION="0.23.4"
+    - env: DISTRIB="ubuntu" PYTHON_VERSION="2.7" CYTHON_VERSION="0.23.5"
```

Comment:
Just a question, is there a good reason why you change this?

Other than this LGTM.

Reply:
oh hmm. actually no, I think I lined up the Cython version at 0.23.5 with the next config without thinking. should we stick with one or the other?

Reply:
Let's keep it the same as the next build, that's fine.

For completeness, we require cython version >= 0.23 in sklearn/_build_utils/__init__.py.